survey/tests.py

```diff
@@ -1,6 +1,36 @@
-from django.test import TestCase
+from django.test import TestCase, Client
+from django.conf import settings
+from django.utils.importlib import import_module
 import models
+import datetime
```

Comment:
Hm, there doesn't seem to be any reason to import the date module anyway---try removing line 6 below?